p10.dts.m4

```diff
@@ -260,6 +260,19 @@ define(`id', eval(`$2 % 2'))dnl
 		reg = <0x00 CONCAT(0x,addr) 0xfffff>;
 		compatible = "ibm,power10-memport";
 		index = <$2>;
+')
+
+dnl
+dnl DIMM([index])
+dnl
+define(`DIMM',
+`
+	CONCAT(dimm@, $1) {
+		#address-cells = <0x02>;
+		#size-cells = <0x01>;
+		reg = <0x00 0x08010000 0x3c00>;
```

Comment:
This is OCMB specific right? Not sure good for DIMM.

Reply:
I think so, How do get the address of DIMM to add in DTS? any idea?

Reply:
This is oly used for SCOM purposed. you already disabled address translation support , i think you can put default data nnow.